lib/reek/configuration/configuration_file_finder.rb

```diff
@@ -17,42 +17,102 @@ class ConfigFileException < StandardError; end
     # The order in which ConfigurationFileFinder tries to find such a
     # configuration file is exactly like above.
     module ConfigurationFileFinder
-      module_function
```

Comment:
Let's go with `class << self` then. That way we'll also never fuck up the visibility (I **always** forget that `class C; private; def self.bar; end; end` will not do what I want).

Deal?